core/templates/dev/head/base.html

```diff
@@ -237,8 +269,8 @@
                     {% endblock %}
                   </div>
                 </div>
-                </div>
               </nav>
+            </div>
```

Comment:
Just to check, are you sure about this? I took a look and it seems like this might just be a superfluous closing div tag that can just be removed (but please double-check this). Otherwise, something's gone wrong somewhere in the indentation, because the indentation here doesn't accord with that two lines below (they should be sibling divs, so the indentation levels should be the same).